help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -61,6 +61,7 @@ Note down the items below before you authenticate to the [!DNL Dynamics 365] CRM
 | `Client ID` | The [!DNL Dynamics 365] Client ID for your [!DNL Azure Active Directory] application. Refer to the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in) for guidance. | `ababbaba-abab-baba-acac-acacacacacac` |
 | `Client Secret` | The [!DNL Dynamics 365] Client Secret for your [!DNL Azure Active Directory] application. You would be using option #2 within the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#authentication-two-options). | `abcde~abcdefghijklmnopqrstuvwxyz12345678` for guidance.|
 | `Tenant ID` | The [!DNL Dynamics 365] Tenant ID for your [!DNL Azure Active Directory] application. Refer to the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in) for guidance.  | `1234567-aaaa-12ab-ba21-1234567890` |
+| `Region` | Region associated with environment URL| If your domain is as below, you need the highlighted value.<br> *org57771b33.`crm`.dynamics.com* |
```

Comment:
This description needs to be improved. Please link out to Microsoft Dynamics docs and explain briefly how depending on where you are provisioned, you need to select a certain region option

Reply:
Yes updated with excepted changes